core/trino-main/src/main/java/io/trino/connector/system/GlobalSystemConnector.java

```diff
@@ -15,7 +15,7 @@
 
 import com.google.common.collect.ImmutableSet;
```

Comment:
update commit message, there is no TableFunctionAdditionalFeaturesProvider any more